> /home/vsts/work/1/s/src/Components/test/E2ETest/ServerExecutionTests/ServerReconnectionWithoutStateTest.cs(14,28): error CS0234: The type or namespace name 'Communication' does not exist in the namespace 'OpenQA.Selenium.BiDi' (are you missing an assembly reference?) [/home/vsts/work/1/s/src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj]
/home/vsts/work/1/s/src/Components/test/E2ETest/ServerRenderingTests/EnhancedNavigationTest.cs(12,28): error CS0234: The type or namespace name 'Communication' does not exist in the namespace 'OpenQA.Selenium.BiDi' (are you missing an assembly reference?) [/home/vsts/work/1/s/src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj]

@copilot please fix

--- a/src/Components/test/E2ETest/ServerExecutionTests/ServerReconnectionWithoutStateTest.cs
+++ b/src/Components/test/E2ETest/ServerExecutionTests/ServerReconnectionWithoutStateTest.cs
@@ -11,7 +11,6 @@ using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
 using Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests;
 using Microsoft.AspNetCore.E2ETesting;
 using OpenQA.Selenium;
-using OpenQA.Selenium.BiDi.Communication;
 using OpenQA.Selenium.DevTools;
 using TestServer;
 using Xunit.Abstractions;

--- a/src/Components/test/E2ETest/ServerRenderingTests/EnhancedNavigationTest.cs
+++ b/src/Components/test/E2ETest/ServerRenderingTests/EnhancedNavigationTest.cs
@@ -9,7 +9,6 @@ using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
 using Microsoft.AspNetCore.E2ETesting;
 using Microsoft.AspNetCore.InternalTesting;
 using OpenQA.Selenium;
-using OpenQA.Selenium.BiDi.Communication;
 using OpenQA.Selenium.DevTools;
 using OpenQA.Selenium.Support.Extensions;
 using TestServer;

